### ISSUE TITLE
Support OpenSearch 2.x

### ISSUE THREAD

**Issue by grignix** [gravitee-io/issues#7804]
## :rainbow: Feature

**I want to** 'Gravitee suport OpenSearch version 2'

in Gravitee 3.15.9 opensearch v2 is not supported, in the logs on the gateway this error:

```
19:20:29.428 [vert.x-worker-thread-1] [] ERROR i.g.r.e.ElasticsearchReporter - OpenSearch version 2.0.0 is not supported by this connector
19:20:29.428 [vert.x-worker-thread-1] [] INFO  i.g.r.e.ElasticsearchReporter - Starting Elastic reporter engine... ERROR
19:20:44.253 [vert.x-eventloop-thread-0] [] ERROR io.vertx.core.impl.ContextImpl - Unhandled exception
java.lang.NullPointerException: Cannot invoke "io.gravitee.reporter.elasticsearch.indexer.Indexer.index(io.gravitee.reporter.api.Reportable)" because "this.indexer" is null
	at io.gravitee.reporter.elasticsearch.ElasticsearchReporter.report(ElasticsearchReporter.java:112)
	at io.gravitee.node.reporter.vertx.eventbus.EventBusReporterWrapper.handle(EventBusReporterWrapper.java:99)
	at io.gravitee.node.reporter.vertx.eventbus.EventBusReporterWrapper.handle(EventBusReporterWrapper.java:33)
	at io.vertx.core.impl.EventLoopContext.emit(EventLoopContext.java:50)
	at io.vertx.core.impl.DuplicatedContext.emit(DuplicatedContext.java:168)
	at io.vertx.core.eventbus.impl.MessageConsumerImpl.dispatch(MessageConsumerImpl.java:177)
	at io.vertx.core.eventbus.impl.HandlerRegistration$InboundDeliveryContext.next(HandlerRegistration.java:169)
	at io.vertx.core.eventbus.impl.HandlerRegistration$InboundDeliveryContext.dispatch(HandlerRegistration.java:134)
	at io.vertx.core.impl.AbstractContext.dispatch(AbstractContext.java:111)
	at io.vertx.core.eventbus.impl.HandlerRegistration.dispatch(HandlerRegistration.java:105)
	at io.vertx.core.eventbus.impl.MessageConsumerImpl.deliver(MessageConsumerImpl.java:183)
	at io.vertx.core.eventbus.impl.MessageConsumerImpl.doReceive(MessageConsumerImpl.java:168)
	at io.vertx.core.eventbus.impl.HandlerRegistration.lambda$receive$0(HandlerRegistration.java:56)
	at io.netty.util.concurrent.AbstractEventExecutor.safeExecute(AbstractEventExecutor.java:164)
	at io.netty.util.concurrent.SingleThreadEventExecutor.runAllTasks(SingleThreadEventExecutor.java:469)
	at io.netty.channel.epoll.EpollEventLoop.run(EpollEventLoop.java:384)
	at io.netty.util.concurrent.SingleThreadEventExecutor$4.run(SingleThreadEventExecutor.java:986)
	at io.netty.util.internal.ThreadExecutorMap$2.run(ThreadExecutorMap.java:74)
	at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30)
	at java.base/java.lang.Thread.run(Unknown Source)
19:20:59.257 [vert.x-eventloop-thread-0] [] ERROR io.vertx.core.impl.ContextImpl - Unhandled exception
```

**Comment by allanian** [comment]
+1

**Comment by gaetanmaisse** [comment]
@grignix I've good news, support of OpenSearch 2 finally landed in APIM. It's available in APIM 3.2010.
We've worked on that at the same time as the ES8 support, I will update the changelog to reflect that.

**Comment by vliska** [comment]
Is that 3.20.1?
